server/auth.go

```diff
@@ -568,6 +590,22 @@ func checkClientTLSCertSubject(c *client, fn func(string) bool) bool {
 		}
 	}
 
+	// Try to get the full RDN Sequence that includes the domain components.
+	var rdns pkix.RDNSequence
+	if _, err := asn1.Unmarshal(cert.RawSubject, &rdns); err == nil {
+		// If found domain components then include roughly following
+		// the order from https://tools.ietf.org/html/rfc2253
+		rdn := cert.Subject.ToRDNSequence().String()
+		dcs := getTLSAuthDCs(&rdns)
+		if len(dcs) > 0 {
+			u := strings.Join([]string{rdn, dcs}, ",")
+			if fn(u) {
+				c.Debugf("Using RDNSequence for auth [%q]", u)
```

Comment:
Could that be a security risk to display those (even if in debug)?

Reply:
think ok since not credentials themselves, but maybe could revisit and change these into being verbose instead

Reply:
That's fine if they are not creds per-se.